stdlib_list/lists/3.9.txt

```diff
@@ -326,6 +356,7 @@ xml.sax
 xml.sax.handler
 xml.sax.saxutils
 xml.sax.xmlreader
```

Comment:
It turns out we did have some submodules in here. I wonder why those were included and the others weren't.